docs/getting-started/getting-started.adoc

```diff
@@ -13,7 +14,8 @@ It is reccomended you have the following installed
  * The latest version of http://openjdk.java.net/install/index.html[OpenJDK]
 
 == Installing
-As of the writing of this guide, the Aerogear Android SDK is not in jcenter and needs to be installed manually.
+
+As of the writing of this guide, the Aerogear Android SDK is not publish anywhere yet and needs to be installed manually.
```

Comment:
How about "is not published in jcenter" instead of "is not published anywhere"

Reply:
Back to the original one!